tests/gnrc_dhcpv6_client_6lbr/Kconfig

```diff
@@ -0,0 +1,8 @@
+if MODULE_ETHOS
+config GNRC_DHCPV6_CLIENT_6LBR_STATIC_ROUTE
+    default y
+config GNRC_NETIF_IPV6_ADDRS_NUMOF
```

Comment:
I'm not sure if you can have multiple definitions for the same config option, but I'm no KConfig expert.